woocommerce/woocommerce-gutenberg-products-block
# Remove unused Add to Cart product element

## What

Remove unused legacy Add to Cart product element. Removed elements:
- block directory
- mention in experimental flag doc
- PHP implementation
- some styles related to the block in other blocks

Fixes https://github.com/woocommerce/woocommerce/issues/44503

## Why

We can remove it because:
- it was [experimental](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/assets/js/atomic/blocks/product-elements/add-to-cart/index.ts#L26)
- it seems untouched for ~3 years
- there’s a new "Add to Cart Form" block that serves the same purpose
- this block was unfinished, [example here](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/assets/js/atomic/blocks/product-elements/add-to-cart/product-types/grouped.tsx)
- this block was not used anywhere. Last usage was in Cross-Sell but it was swapped with Product Button: https://github.com/woocommerce/woocommerce-blocks/pull/8914

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Please double-check the potential usage of this block across repo. Keywords I used among others (please try to invent your own for better coverage):
  - `add-to-cart`
  - `ProductAddToCart`
2. Smoke test blocks
3. Make sure project builds correctly and all tests are passing


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Add suggested changelog entry here.
